### ISSUE TITLE
Extra commas are considered valid

### ISSUE THREAD

**Issue by Enumer8** [RicoSuter/NJsonSchema#942]
I'm using the latest NJsonSchema NuGet package version 9.13.28.

NJsonSchema catches some JSON errors (for example, two commas in a row), but it does not catch the case where there is an extra, trailing comma. For example:

```
   static void Main()
   {
      var jsonSchemaText =
@"{
  ""$schema"": ""http://json-schema.org/draft-07/schema#"",
  ""type"": ""object"",
  ""title"": ""Test"",
  ""properties"": {
    ""hello"": {
      ""type"": ""string""
    }
  }
}";
      var jsonText =
@"{
  ""hello"": ""world"",
}";
      var schema = JsonSchema4.FromJsonAsync(jsonSchemaText).Result;
      var errors = schema.Validate(jsonText);
   }
```
Notice the extra comma after "world". NJsonSchema does not report any errors and doesn't throw an exception. https://jsonschemalint.com, for example, correctly reports that the JSON document is invalid. Is there a way to enable a "strict" JSON mode in NJsonSchema? This is important, because having the extra comma in the JSON string will crash the Python parser, for instance.

**Comment by RicoSuter** [comment]
We are using Newtonsoft.Jsoj for deserialization... we need to look there if it supports strict parsing. Do you know of such a setting?

**Comment by akshaybabloo** [comment]
I haven't tried this but I think they support trailing commas https://github.com/JamesNK/Newtonsoft.Json/issues/818
